[2024-10-27T09:12:10.613+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bot_dag.run_bot_notebook scheduled__2024-10-27T09:02:00+00:00 [queued]>
[2024-10-27T09:12:10.623+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bot_dag.run_bot_notebook scheduled__2024-10-27T09:02:00+00:00 [queued]>
[2024-10-27T09:12:10.624+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-10-27T09:12:10.793+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): run_bot_notebook> on 2024-10-27 09:02:00+00:00
[2024-10-27T09:12:10.796+0000] {standard_task_runner.py:57} INFO - Started process 35936 to run task
[2024-10-27T09:12:10.803+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'bot_dag', 'run_bot_notebook', 'scheduled__2024-10-27T09:02:00+00:00', '--job-id', '6715', '--raw', '--subdir', 'DAGS_FOLDER/bot_dag.py', '--cfg-path', '/tmp/tmpxbh20apd']
[2024-10-27T09:12:10.805+0000] {standard_task_runner.py:85} INFO - Job 6715: Subtask run_bot_notebook
[2024-10-27T09:12:10.983+0000] {task_command.py:410} INFO - Running <TaskInstance: bot_dag.run_bot_notebook scheduled__2024-10-27T09:02:00+00:00 [running]> on host 454d1fc6eddf
[2024-10-27T09:12:11.153+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bot_dag' AIRFLOW_CTX_TASK_ID='run_bot_notebook' AIRFLOW_CTX_EXECUTION_DATE='2024-10-27T09:02:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-27T09:02:00+00:00'
[2024-10-27T09:12:11.154+0000] {bot_dag.py:9} INFO - Функция run_bot_notebook запущена
[2024-10-27T09:12:11.225+0000] {bot_dag.py:22} ERROR - Ошибка при запуске скрипта: python: can't open file '/src/pbot.py': [Errno 2] No such file or directory

[2024-10-27T09:12:11.226+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-10-27T09:12:11.240+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=bot_dag, task_id=run_bot_notebook, execution_date=20241027T090200, start_date=20241027T091210, end_date=20241027T091211
[2024-10-27T09:12:11.266+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-10-27T09:12:11.285+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
